bip-taro-proof-file.mediawiki

```diff
@@ -124,6 +124,9 @@ blob with the following format:
 * type: 9 (<code>taro_input_splits</code>)
 ** value:
 *** [<code>...*byte</code>:<code>nested_proof_map</code>]
+* type: 10 (<code>asset_group_key_reveal</code>)
+** value: 
+*** [<code>64*byte</code>:<code>pub_key || asset_script_root</code>]
```

Comment:
I think only the raw public key is needed? Since with that you can get to the internal key, then you'd force (?) an initial script path reveal so the internal key is known.

Reply:
The idea here was that if keypath spend is used with the group key, then the taproot is not revealed in the witness (only a signature), which is why we add it here. Otherwise I don't think one could actually verify the group key derivation in that case?